addons/event/views/event_tag_views.xml

```diff
@@ -72,7 +72,8 @@
                     <sheet>
                         <group>
                             <field name="name"/>
-                            <field name="category_id" widget="many2one"/>
+                            <field name="event_tag_website_id" invisible="1"/>
```

Comment:
As this field is defined in website_event, it should be done with a view override in website_event. Otherwise installing event will crash (hence the red runbot). Purpose is to have a modular approach: each sub addon adds its own features and fields, and everything should be contained in this module. Base event module is website agnostic, this model does not exist until website(_event) is installed.

Reply:
@tde-banana-odoo thanks for noting, i don't know how i missed it (actually i always delete database and install it again, seems i missed something)